other/bootstrap_daemon/README.md

```diff
@@ -1,27 +1,25 @@
-##Instructions for Debian
+##Instructions 
 
 For security reasons we run the daemon under its own user.
 Create a new user by executing the following:
 ```sh
-sudo useradd --system --shell /sbin/nologin --comment "Account to run Tox's DHT bootstrap daemon" --user-group tox-bootstrapd
+sudo useradd --home-dir /var/lib/tox-bootstrapd --create-home --system --shell /sbin/nologin --comment "Account to run Tox's DHT bootstrap daemon" --user-group tox-bootstrapd
```

Comment:
> useradd --home-dir /var/lib/tox-bootstrapd --create-home

``/var/lib`` isn't actually required in a distro, and may break stuff creating the home.